src/parser/jobs/pld/modules/OGCDDowntime.tsx

```diff
@@ -1,24 +1,17 @@
 import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 
-// Allowed downtime set to 4s to account for PLD's natural rotation drift.
-const ALLOWED_DOWNTIME_FOF = 4000
-const ALLOWED_DOWNTIME_REQ = 4000
+// Allowed downtime set to 2s to account for PLD's natural rotation drift.
+const ALLOWED_DOWNTIME_FOF = 3000
+const ALLOWED_DOWNTIME_REQ = 2000
```

Comment:
Comment here indicates average allowed downtime of 2s, but the constant for Fight or Flight downtime is 3s.  Is that intended?